wheel/generate_type_stubs.py

```diff
@@ -305,6 +305,22 @@ def confirm_not_included_already_hashed(
     proof: bytes,
 ) -> bool: ...
 
+def validate_clvm_and_signature(
+    new_spend: SpendBundle,
+    max_cost: int,
+    constants: ConsensusConstants,
+    peak_height: int,
+    cache: Optional[BLSCache],
+) -> Tuple[SpendBundleConditions, float]: ...
+
+def get_name_puzzle_conditions(
```

Comment:
you need to export the `get_flags_for_height_and_constants()` as well. We need that function on the python side.